<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="Panel" name="m_gamePanel" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>50</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>450</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_hostButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_regretButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Regret</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_info">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Panel</class>
   <extends>QWidget</extends>
   <header>Panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
